pkg/ddevapp/global_dotddev_assets/commands/host/self-upgrade

```diff
@@ -12,7 +12,7 @@ case $mypath in
   "/usr/bin/ddev")
     if [[ ${OSTYPE} = "linux-gnu"* ]]; then
       if command -v apt; then echo "You seem to have an apt-installed ddev, upgrade with 'sudo apt update && sudo apt upgrade -y ddev'";
-      elif [ -f /etc/arch-release ] && command -v yay >/dev/null ; then echo "You seem to have yay-installed ddev (AUR), upgrade with 'yay -Syu ddev'";
+      elif [ -f /etc/arch-release ] && command -v yay >/dev/null ; then echo "You seem to have yay-installed ddev (AUR), upgrade with 'yay -Syu ddev-bin'";
```

Comment:
This one is a functionality change and requires manual testing.

Reply:
Here it is:

```
# there is not such package
$ yay -Syu ddev
...
 -> Could not find all required packages:
        ddev (Target)

# search for ddev-bin package
$ yay -Ss ddev-bin
aur/ddev-bin 1.21.4-1 (+11 0.37) (Installed)
    DDEV: a local web development environment

# upgrade ddev-bin
$ yay -Syu ddev-bin
...
==> Making package: ddev-bin 1.21.4-1
...
```